pkg/cableengine/healthchecker/pinger.go

```diff
@@ -111,8 +114,11 @@ func (p *pingerInfo) Start() {
 			default:
 				err := p.doPing()
 				if err != nil {
-					klog.Errorf("Unable to start pinger for IP %q: %v", p.ip, err)
-					return
+					klog.Errorf("Unable to start pinger for IP %q: %v, retry in %d seconds", p.ip, err, doPingRetryTimeout)
+					time.Sleep(doPingRetryTimeout * time.Second)
+				} else {
+					klog.Infof("Successfully started pinger for IP %q ", p.ip)
+					break
```

Comment:
I think we should make the minimal change here and just retry (ie not return) on failure. A nil return from `doPing` means it ran to completion so the log message here isn't meaningful. Also let L112 above decide to exit the loop instead of breaking here.

Reply:
Sounds reasonable, do we report the pinger status(working/not working) some cr status?